### PR TITLE
update link object yaml file

### PR DESCRIPTION
**Pull request by dthiex**
Update yaml file to give more meaningful data in stac json link object.

### CODE REVIEW

File: collections/sentinel-2-l2a.yaml [hunk 1]
```diff
@@ -54,60 +54,80 @@ CustomScripts:
...
+    title: "Evalscript to generate False Color Urban imagery"
+    sentinelhub:layer_name: "False Color Urban"
+    sentinelhub:mosaicking_order: "mostRecent"
+    sentinelhub:upsampling: "BICUBIC"
+  - href: "https://custom-scripts.sentinel-hub.com/custom-scripts/sentinel-2/false-color-urban-rgb/script.js"
+    rel: "processing-expression"
+    type: "application/javascript"
+    title: "Evalscript to generate False Color Urban imagery"
+    sentinelhub:layer_name: "False Color Urban"
+    sentinelhub:mosaicking_order: "mostRecent"
```
Comment (zcernigoj) [inline collections/sentinel-2-l2a.yaml:94]: These two are duplicated

File: collections/alos-2-palsar-2-scansar-for-agriculture.yaml [hunk 2]
```diff
@@ -50,19 +50,25 @@ Resources:
...
+    title: "Details about running Evalscripts"
+  - href: "https://raw.githubusercontent.com/eurodatacube/public-collections/main/collections/alos-2-palsar-2-scansar-for-agriculture/hh_script.js"
+    rel: "processing-expression"
+    type: "application/javascript"
+    title: "Evalscript to generate HH imagery"
+    sentinelhub:layer_name: "HH"
+    sentinelhub:mosaicking_order: "mostRecent"
+    sentinelhub:upsampling: "BICUBIC"
+  - href: "https://raw.githubusercontent.com/eurodatacube/public-collections/main/collections/alos-2-palsar-2-scansar-for-agriculture/hv_script.js"
+    rel: "processing-expression    "
```
[the remark this answers is not in the document]
Comment (zcernigoj) [reply collections/alos-2-palsar-2-scansar-for-agriculture.yaml]: Thanks, removed

File: collections/landsat-1-5-mss-l1.yaml [hunk 3]
```diff
@@ -46,30 +46,38 @@ CustomScripts:
...
+    title: "Evalscript to generate NDVI imagery"
+    sentinelhub:layer_name: NDVI
+    sentinelhub:mosaicking_order: mostRecent
+    sentinelhub:upsampling: NEAREST
+  - href: "https://custom-scripts.sentinel-hub.com/landsat-1-5-mss/ndwi/script.js"
     rel: processing-expression
+    type: "application/javascript"
+    title: "Evalscript to generate NDWI imagery"
+    sentinelhub:layer_name: "NDWI"
+    sentinelhub:mosaicking_order: mostRecent
```
[the remark this answers is not in the document]
Comment (zcernigoj) [reply collections/landsat-1-5-mss-l1.yaml]: Nice catch, added

File: collections/sentinel-1-grd.yaml [hunk 4]
```diff
@@ -62,25 +62,31 @@ Resources:
...
+  - href: "https://custom-scripts.sentinel-hub.com/sentinel-1/sar_false_color_visualization-2/script.js"
     rel: processing-expression
+    type: "application/javascript"
+    title: "Evalscript to generate Enhanced visualization imagery"
+    sentinelhub:layer_name: Enhanced visualization
+    sentinelhub:acquisition_mode: IW
+    sentinelhub:polarization: DV
+    sentinelhub:resolution: HIGH
+    sentinelhub:mosaicking_order: mostRecent
+    sentinelhub:upsampling: BICUBIC
```
[the remark this answers is not in the document]
Comment (zcernigoj) [reply collections/sentinel-1-grd.yaml]: True, thanks. Added

File: collections/sentinel-5p-l2-tropomi.yaml [hunk 5]
```diff
@@ -59,67 +59,81 @@ Resources:
...
+    sentinelhub:mosaicking_order: "mostRecent"
+    sentinelhub:upsampling: "BICUBIC"
+  - href: "https://custom-scripts.sentinel-hub.com/custom-scripts/sentinel-5p/formaldehyde/script.js"
+    rel: "processing-expression"
+    type: "application/javascript"
+    title: "Evalscript to generate Formaldehyde - HCHO imagery"
+    sentinelhub:layer_name: "Formaldehyde - HCHO"
+    sentinelhub:mosaicking_order: "mostRecent"
+    sentinelhub:upsampling: "BICUBIC"
+  - href: "https://custom-scripts.sentinel-hub.com/custom-scripts/sentinel-5p/formaldehyde/script.js"
```
[the remark this answers is not in the document]
Comment (zcernigoj) [reply collections/sentinel-5p-l2-tropomi.yaml]: Ha, noticed the one in `sentinel-2-l2a.yaml` (first comment) but not this one. Thanks, removed.

File: collections/alos-2-palsar-2-scansar-for-agriculture.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (dthiex) [reply collections/alos-2-palsar-2-scansar-for-agriculture.yaml]: 👍

File: collections/landsat-1-5-mss-l1.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (dthiex) [reply collections/landsat-1-5-mss-l1.yaml]: 👍

File: collections/sentinel-1-grd.yaml (hunk 4 above)
[the remark this answers is not in the document]
Comment (dthiex) [reply collections/sentinel-1-grd.yaml]: 👍

File: collections/sentinel-5p-l2-tropomi.yaml (hunk 5 above)
[the remark this answers is not in the document]
Comment (dthiex) [reply collections/sentinel-5p-l2-tropomi.yaml]: 👍

**Pull request merged**